libs/gi/page-team/src/TeamSetting/TeamDelModal.tsx

```diff
@@ -0,0 +1,175 @@
+      <CardThemed>
+        <CardHeader
+          title={
+            <Box
+              sx={{
+                display: 'flex',
+                gap: 1,
+                alignItems: 'center',
+              }}
+            >
+              <Box>Delete Team:</Box>
+              <strong>{name}</strong>
+              {description && (
+                <BootstrapTooltip title={description}>
+                  <InfoIcon />
+                </BootstrapTooltip>
+              )}
+            </Box>
+          }
+          action={
+            <IconButton onClick={onHide}>
+              <CloseIcon />
+            </IconButton>
+          }
+        />
+        <Divider />
+        <CardContent sx={{ display: 'flex', flexDirection: 'column', gap: 1 }}>
+          <Alert severity="info">
+            Removing the team will remove: resonance buffs, and enemy configs
+            stored in the team. Loadouts that are only in this team are also
+            selected by default for deletion.
+          </Alert>
+          {loadoutData.map((loadoutDatum, i) =>
+            loadoutDatum ? (
+              <LoadoutDisplay
+                key={i}
+                teamCharId={loadoutDatum.teamCharId}
+                selected={delArr.includes(i)}
+                onClick={() => setDelArry((arr) => toggleArr(arr, i))}
+                inTeams={inTeamsData[i]}
+              />
+            ) : null
+          )}
+        </CardContent>
+        <Divider />
+        <CardContent sx={{ display: 'flex', justifyContent: 'flex-end' }}>
+          <Button
+            color="error"
+            startIcon={<DeleteForeverIcon />}
+            onClick={onDelete}
+          >
+            Delete
+          </Button>
+        </CardContent>
+      </CardThemed>
+    </ModalWrapper>
+  )
+}
+function LoadoutDisplay({
+  teamCharId,
+  selected,
+  onClick,
+  inTeams,
+}: {
+  teamCharId: string
+  selected: boolean
+  onClick: () => void
+  inTeams: Team[]
+}) {
+  const teamChar = useTeamChar(teamCharId)!
+  const { key: characterKey } = teamChar
+  const teamData = useCharData(characterKey)
+  const { target: charUIData } = teamData?.[characterKey] ?? {}
+
+  const dataContextValue: dataContextObj | undefined = useMemo(() => {
+    if (!teamData || !charUIData) return undefined
+    return {
+      data: charUIData,
+      teamData,
+      compareData: undefined,
+    }
+  }, [charUIData, teamData])
+  if (!dataContextValue) return
+  return (
+    <DataContext.Provider value={dataContextValue}>
+      <CardThemed
+        bgt="light"
+        sx={{ border: selected ? '2px red solid' : undefined }}
+      >
+        <CardActionArea onClick={onClick}>
+          <LoadoutHeaderContent teamCharId={teamCharId}>
+            <ColorText color={inTeams.length === 1 ? 'success' : 'warning'}>
+              <Typography>
+                {inTeams.length === 1
+                  ? 'Only in current team'
+                  : `In ${inTeams.length} teams`}
```

Comment:
Tooltip for team names? Or too much clutter?

Reply:
probably, I think the value here is to indicate if loadouts are in more than one team or not, so not 100% sure about showing all of them yet.